pkg/agent/flowexporter/connections/interface.go

```diff
@@ -22,5 +22,6 @@ import (
 // netfilter socket (OVS kernel datapath) and ovs-appctl command (OVS userspace datapath).
 // In future, support will be extended to Windows.
 type ConnTrackDumper interface {
-	DumpFlows(zoneFilter uint16) ([]*flowexporter.Connection, error)
+	DumpFlows(zoneFilter uint16) ([]*flowexporter.Connection, int, error)
+	GetMaxConnections() (int, error)
```

Comment:
Interface methods should have a comment with a description of the returned parameters

Reply:
Thanks, done.